packages/core/src/contrib/__tests__/Minkowski.test.ts

```diff
@@ -125,39 +125,39 @@ describe("convexPartitions", () => {
 
   test("Figure 1 from [HM83]", () => {
     // https://link.springer.com/content/pdf/10.1007/3-540-12689-9_105.pdf
-    // point locations approximated by tracing in Inkscape, then y-coordinates
-    // are inverted because screen coordinates are upside down compared to math,
-    // and the library we use expects the polygon to be counterclockwise
+    // point locations approximated by tracing in Inkscape; also note, this
+    // polygon is actually clockwise because screen coordinates are upside down
+    // compared to math
     const p = [
-      [-316.39758, 117.40885],
```

Comment:
Maybe we can leave this as well and test both orientations.

Reply:
Hmm we could, but the other test is counterclockwise so we kind of already are testing both. Up to you though!

Reply:
It is now, but both tests will be a bit more future proof I think

Reply:
Fair enough! Done.